documentation/rom_simulation/0d-solver/user-guide-run/readme.md

```diff
@@ -28,15 +28,15 @@ in the test cases at `svZeroDSolver/tests/test_interface`.
 #### In Python
 
 Please make sure that
-you installed svZerodSolver via pip to enable this feature. We start by
+you have installed svZerodSolver via pip to enable this feature. We start by
 importing pysvzerod:
 
 ```python
 >>> import pysvzerod
 ```
 
 Next, we create a solver from our configuration. The configuration can
-be specified by either a path to a JSON file:
+be specified by either a path to a JSON file. In this case we are using the test case ```steadyFlow_RLC_R.json```. if you have are running this code from the top level of the svZeroDSolver directory, use the following path. Otherwise, adjust the path to reflect the directory you are currently in.
```

Comment:
You have an `either` but where is the `or` ? 

There are also spelling mistakes `if`.